package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "engines": {
```

Comment:
I'm just wondering why didn't you remove all Heroku-related stuff BEFORE working on new feature?